packages/firebase_auth/README.md

```diff
@@ -1,14 +1,126 @@
-# Firebase Authentication for Flutter
+# firebase_auth plugin
+A Flutter plugin to use the [Firebase Authentication API](https://firebase.google.com/products/auth/).
 
 [![pub package](https://img.shields.io/pub/v/firebase_auth.svg)](https://pub.dartlang.org/packages/firebase_auth)
 
-A Flutter plugin to use the [Firebase Authentication API](https://firebase.google.com/products/auth/).
-
 *Note*: This plugin is still under development, and some APIs might not be available yet. [Feedback](https://github.com/flutter/flutter/issues) and [Pull Requests](https://github.com/flutter/plugins/pulls) are most welcome!
 
 ## Usage
-To use this plugin, add `firebase_auth` as a [dependency in your pubspec.yaml file](https://flutter.io/platform-plugins/).
 
-## Getting Started
+### Import dependent package
+First install the dependency. Follow the [Google sign-in plugin installation instructions](https://pub.dartlang.org/packages/google_sign_in#pub-pkg-tab-installing).
```

Comment:
By the way, this is dependent on the pub getting updated. I see the change hasn't been pushed out yet.